TEST

```diff
@@ -5,25 +5,25 @@
 #TEST_PREPARE_SHARES="yes"
 
 ### SPLAT
-#TEST_SPLAT_SKIP="yes"
+TEST_SPLAT_SKIP="yes"
```

Comment:
are the changes to TEST appropriate for everyone?  methinks maybe this is a temporary change and not intended for broad adoption?

Reply:
@richardelling These changes aren't going to be permanent. I just wanted to restrict the test suite to my new tests. Trying to figure out what I forgot to do to allow `test-runner` to accept my test.

Reply:
When you next refresh this can you drop the changes to TEST so we can a full test run.